### ISSUE TITLE
2 views from remove video

### ISSUE THREAD

**Issue by nasersobhan** [andyet/SimpleWebRTC#722]
I want to have 2 videos from each incoming video. I want to have a list of all videos including my own screen in the side in small box and when i click on one of them it should move to other side with bigger size. 
I tried to move it but it removes the small one and I can't move back to what it was. 

can anyone help me?

**Comment by hokuco** [comment]
without providing code/a screenshot, I don't know what you really want to do but I think styling the video element you want to hide like below would be a great idea:
`.hiddenvideo{display:none}`
then to show it again:
`.shownvideo{display:inline}`

**Comment by nasersobhan** [comment]
no, I know css, I use the normal code in the github noting special but I am trying to make 2 videos from one connection (one Small and one big)  or at least something that moves the video src from one video to another video element. so to be short, I want two video elements with one source how can I get src from current video element and move it to another.

**Comment by hokuco** [comment]
oh, that makes more sense, I can't help much but I think you can actually grab frames from the localvideo, or more simply, you could use 2 iframes on a seperate video page

**Comment by wardhanster** [comment]
You can use html5 canvas API to do that 
take a look at 
https://www.w3schools.com/tags/canvas_drawimage.asp for a quick and easy demo.
